examples/00-mapdl-examples/bracket_static.py

```diff
@@ -0,0 +1,743 @@
+# it to generate the fillet line.
+# That is done using the :meth:`mapdl.get() <ansys.mapdl.core.Mapdl.get>` command
+# from PyMAPDL or `GET` in pymapdl.
+#
+# Since we have selected one line, we can use the ``MAX`` and ``NUM`` arguments
+# for the :meth:`mapdl.get() <ansys.mapdl.core.Mapdl.get>` command.
+#
+# Select first line for fillet
+line1 = mapdl.lsel("S", "LOC", "Y", box1[2])
+l1 = mapdl.get("line1", "LINE", 0, "NUM", "MAX")
+
+
+###############################################################################
+#
+# If we write the command to a python parameter (``line1``) we can use either
+# the APDL parameter ``l1`` or the python parameter ``line1`` when we create
+# the fillet line.
+#
+# Select second line for fillet, create python parameter
+line2 = mapdl.lsel("S", "LOC", "X", box2[0])
+l2 = mapdl.get("line2", "LINE", 0, "NUM", "MAX")
+
+###############################################################################
+# Once we have both lines selected we can use the APDL command
+# :meth:`mapdl.lfillt() <ansys.mapdl.core.Mapdl.lfillt>` to generate the fillet
+# between the lines.
+#
+# **Note** that python could return a list if more than one line is selected.
+
+###############################################################################
+# Here we use a mix of the APDL parameter as a string ``line1`` and the ``l2``
+# python parameter to create the fillet line.
+#
+# Create fillet line using selected line (parameter names)
+fillet_radius = 0.4
+mapdl.allsel()
+line3 = mapdl.lfillt("line1", l2, fillet_radius)
+
+mapdl.allsel()
+mapdl.lplot(vtk=True, cpos="xy")
+
+###############################################################################
+# Create fillet area
+# ~~~~~~~~~~~~~~~~~~
+#
+# To create the area delineated by ``line1``, ``line2`` and newly created
+# ``line3``, we use the :meth:`mapdl.al() <ansys.mapdl.core.Mapdl.al>` command.
+# The three lines are the input. If we select them all, we can use the ``'ALL'``
+# argument to create the area.
+#
+# First we have to reselect the newly created lines in the fillet area.
+# To do this we can use the fillet_radius parameter and the
+# :meth:`mapdl.lsel() <ansys.mapdl.core.Mapdl.lsel>` command.
+#
+# For the two newly created straight lines, the length will be the same as the
+# fillet_radius. So we can use the length argument with the
+# :meth:`mapdl.lsel() <ansys.mapdl.core.Mapdl.lsel>` command.
+#
+
+mapdl.allsel()
+
+# Select lines for the area
+mapdl.lsel("S", "LENGTH", "", fillet_radius)
+
+###############################################################################
+# Additionally, we need the fillet line itself (``line3``) but we can use the
+# :meth:`mapdl.lsel() <ansys.mapdl.core.Mapdl.lsel>` command again with either
+# the ``'RADIUS'`` argument if there is only one line with that radius in our model
+# or more directly using the parameter name of the line.
+# Note the ``'A'`` to additionally select items.
```

Comment:
```suggestion
# Additionally, you need to get the fillet line itself (``line3``). You can use the
# :meth:`mapdl.lsel() <ansys.mapdl.core.Mapdl.lsel>` command again with either
# the ``'RADIUS'`` argument if there is only one line with that radius in the model
# or more directly use the parameter name of the line.
# Note the ``'A'`` to additionally select items.
```
The earlier Note you bolded. I'd prefer it wasn't bolded. However, they should be consistently formatted within this example.

Reply:
Done.